src/fetcher/create-breadcrumbs.tsx

```diff
@@ -1,6 +1,5 @@
 // TODO: needs type declaration
 export function createBreadcrumbs(uuid: any) {
-  if (!uuid || !uuid.navigation || !uuid.navigation === undefined) return []
```

Comment:
`!uuid.navigation === undefined` doesn't make sense since you already check `!uuid.navigation`. Also, the GraphqL API should return `null` for option values (not `undefined`)

Reply:
That's the reason while I deleted this line, we check for navigation later on.